recipes/pyopenms/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "pyopenms" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower  }}
+  version: {{ version  }}
+
+source:
+  - url: https://github.com/OpenMS/OpenMS/releases/download/Release{{ version }}/OpenMS-{{ version }}-src.tar.gz
+    sha256: 86e8176b0f89cc86a10f5c3e60c34140d591e9e6c85f129ccbfa13ecb16a9af2 
+  - url: http://ftp.mi.fu-berlin.de/pub/OpenMS/contrib-sources/WildMagic5.tar.gz
+    sha256: cbdc9fa23880070b46b63764eb34795e25d099d2d06c259ad58c21d7bea45772
+    folder: wildmagic
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - autoconf
+    - automake
+  host:
+    - python
+    - autowrap
+#    - nose
+#    - wheel
+    - numpy
+    - pip
+#    - mesalib
+#    - zlib
+#    - libtool
+#    - xerces-c
+#    - gsl
+#    - boost
+#    - eigen
+#    - glpk
+#    - bzip2
+#    - qt
+#    - libsvm
+    - seqan-library ==1.4.2
+#    - coinmp
+    - openms {{ version }}
+  run:
+    - python
+    - autowrap
+#    - nose
+#    - wheel
+    - numpy
+#    - pip
+#    - mesalib
+#    - zlib
+#    - xerces-c
+#    - gsl
+#    - boost
+#    - eigen
+#    - glpk
+#    - bzip2
+#    - qt
+#    - libsvm
+#    - coinmp
+    - openms {{ version }}
+
+test:
+  commands:
+    - OPENMS_DATA_PATH=${PREFIX}/share/OpenMS/ FileMerger --help
```

Comment:
maybe replace this with running and importing pyOpenMS ? Or even something as easy as writing and reading a mzML file on disk?

Reply:
Would ```python -c "import pyopenms; print(pyopenms.__version__)"``` work as a test?

Reply:
it should, it will test whether it was built and can be imported